core/src/main/kotlin/org/evomaster/core/problem/graphql/builder/StateBuilder.kt

```diff
@@ -18,6 +19,11 @@ object StateBuilder {
         "__Field", "__InputValue", "__Type", "__TypeKind"
     )
 
+    /**
+     * Used to create unique IDs
+     */
+    private val idGenerator = AtomicInteger()
```

Comment:
this must be avoided, ie mutable static state. 
if really need it as static state, rather use `StaticCounter`

Reply:
Done.